### DISCUSSION TITLE
Header back title when previous screen title is too long.

### DISCUSSION THREAD

**Discussion by LazyAfternoons** [react-navigation/react-navigation#10865] [Q&A] [answered]
Hello, I'm currently using a NativeStackNavigator in my application and I don't understand how to customize the header back title only when the previous screen title is too long.
Passing ```headerBackTitle: 'Test'``` in screenOptions doesn't seem to work, it defaults the value to 'Test' even when the title fits the available space. 
Am I missing something? Thanks.

**Comment by satya164** [comment, marked as answer]
Native Stack doesn't support such customization. Like the documentation for `headerBackTitle` says:

> Title string used by the back button on iOS. Defaults to the previous scene's title, or "Back" if there's not enough space.

It doesn't work like how you're expecting the option to work.

https://reactnavigation.org/docs/native-stack-navigator/#headerbacktitle

**Reply by LazyAfternoons** [reply]
> Native Stack doesn't support such customization. Like the documentation for `headerBackTitle` says:
> 
> > Title string used by the back button on iOS. Defaults to the previous scene's title, or "Back" if there's not enough space.
> 
> It doesn't work like how you're expecting the option to work.
> 
> https://reactnavigation.org/docs/native-stack-navigator/#headerbacktitle

Thanks for your help. Is there a way to localize that "Back" screen then? It seems to be always in english, no matter what.

**Reply by satya164** [reply]
You can open an issue in react-native-screens repo since the implementation lives there
